### DISCUSSION TITLE
Lession 12 - ERC20Mock Problem because OpenZeppelin Contracts Updated

### DISCUSSION THREAD

**Discussion by terranweb3** [Cyfrin/foundry-full-course-cu#524] [Q&A] [answered]
Hi, I have an ERC20 problem in my code that makes it impossible for me to follow the instructions in lesson 12.

In post 12 at this time: [https://youtu.be/wUjYK5gwNZs?t=6972](https://youtu.be/wUjYK5gwNZs?t=6972)

In the video, at the path openzeppelin-contracts/contracts/mocks/ERC20Mock.sol, you need to enter 4 arguments to get the mock token.

But, in my current project, the openzeppelin library has changed in this file and it looks like this:
`
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.0;

import {ERC20} from "../token/ERC20/ERC20.sol";

contract ERC20Mock is ERC20 {
    constructor() ERC20("ERC20Mock", "E20M") {}

    function mint(address account, uint256 amount) external {
        _mint(account, amount);
    }

    function burn(address account, uint256 amount) external {
        _burn(account, amount);
    }
}
`
You can also view this file at: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/mocks/token/ERC20Mock.sol

I have searched many ways for my problem but all failed, is there any way to solve this problem?
Thank you very much for reading this issue and answering it.

**Comment by Mepsss** [comment, marked as answer]
You can go to the github repo of lesson 12, lib/contracts/mocks/ERC20Mock.sol and copy the code into your Mock in your library and use  the old Mock that Patrick used. Then you have 4 arguments like he does

**Reply by S23Web3** [reply]
That contract does not exist anymore, openzeppelin updated their library, need to have a solution
